### ISSUE TITLE
outputdir in oxdoc.xml

### ISSUE THREAD

**Issue by linc0380** [yozw/oxdoc#13]
There are the contents of my `oxdoc.xml`:
```
<oxdoc>
  <option formulas="mathjax"/>
  <option internals="off"/>
  <option icons="on"/>
  <option outputdir="Oxdoc"/>
  <option latexpackages="unicode-math,mathtools"/>
</oxdoc>
```
The file is in the same directory as the `ox` file processed by `oxdoc`. Yet the documentation is saved in a subdirectory named `doc`.

**Comment by yozw** [comment]
It turns out that your `oxdoc.xml` doesn't use the right syntax. It should have read:
```
<oxdoc>
  <option name="formulas" value="mathjax" />
</oxdoc>
```

Your options were entirely ignored. 

I have no idea why I ended up with two different conventions... :) 

In https://github.com/yozw/oxdoc/commit/966d0b8ee17f792f938024aca4c5680ae6dedcb9, I have added support for both conventions, and errors in config.xml now cause oxdoc to crash with a clear error message. Note that "internals" in your config should be "showinternals".
